release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -646,16 +646,17 @@ Some features in this release are currently in Technology Preview. These experim
 
 link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
 
-In the table below, features are marked with the following statuses:
+In the following tables, features are marked with the following statuses:
 
 * *TP*: _Technology Preview_
 * *GA*: _General Availability_
 * *-*: _Not Available_
 * *DEP*: _Deprecated_
 
-//TODO: remove anything that has been GA since 4.9?
+[id="ocp-networking-technology-preview-features"]
+=== Networking technology preview features
 
-.Technology Preview tracker
+.Networking technology Preview tracker
```

Comment:
Is there a particular order for these tables? Just wondering if we should do a purposeful order... alphabetical is an option, but not sure if it's the best. However, I'm assuming that most, if not all, of these categories match categories in the new feature section. What if we mirrored that same order here with these?

Reply:
related: afaik there is not really rhyme or reason for the order of the new features (and bugfixes) sections either. I think I had on my RN enhancement wishlist to revisit these, figure out what order makes sense, and make sure they match. (just sharing my thoughts, this is def out of scope for the PR imho)

Reply:
Yeah def don't need to be worrying about that feature category order in this PR, more just to consider matching this table to that.

However I actually vaguely remember someone saying that they had tried to reorder the categories so that they were kind of in line with our topic map TOC on the left (maybe Cody?). And when I check just now, it actually does kind of follow that for a lot of them for the ones that match top-level books. (There might need to be a few adjustments, like moving the auth category up to below "Security" and some others). So maybe we could consider moving those around to be in line.

Can't speak to the bug fixes section though :wink:

Reply:
Will revisit this comment later. I think it would be great to have them in an intentional order, just need to figure out what that would look like!